test/powershell/Modules/Microsoft.PowerShell.Management/Get-HotFix.Tests.ps1

```diff
@@ -58,4 +58,9 @@ Describe "Get-HotFix Tests" -Tag CI {
         $hotfixes = Get-HotFix -ComputerName localhost
         $hotfixes.Count | Should -Be $qfe.Count
     }
+
+    It "Get-Hotfix can accept ComputerName via pipeline" {
+        { [PSCustomObject]@{ComputerName = 'UnavailableComputer'} | Get-HotFix } |Should -Throw -ExpectedMessage "The RPC server is unavailable"
```

Comment:
Please use `-ErrorId` instead of localization sensitive `-ExpectedMessage`

Reply:
yeah !, corrected now.